apps/AnyDesk/install-64

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+version=6.3.0-1
+
+function check-armhf() {
+	ARMHF="$(dpkg --print-foreign-architectures | grep "armhf")"
+}
+
+#add armhf architecture (multiarch)
+check-armhf
+if [[ "$ARMHF" == *"armhf"* ]]; then
+  echo "armhf arcitecture already added..."
+else
+  sudo dpkg --add-architecture armhf
+  check-armhf
+  if [[ "$ARMHF" != *"armhf"* ]]; then
+    error "armhf architecture should be added by now, but it isn't!"
+  fi
+fi
+
+unset rpi_arm_userspace
+# only install the libraspberrypi0 arm32 package if the user already has the libraspberrypi0 arm64 package installed
+if package_installed libraspberrypi0 ; then
+  rpi_arm_userspace="libraspberrypi0:armhf"
+fi
```

Comment:
the deprecated libraspberrypi0 libraries are required to run anydesk
```
libbcm_host.so
libvcos.so
libvchiq_arm.so
libbrcmGLESv2.so
libbrcmEGL.so
```
this needs to be changed to check if `libraspberrypi0:armhf` exists and if available always install it. otherwise exit the script.